Calculator/TestAddition.cs

```diff
@@ -0,0 +1,38 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Calculator
+{
+    [TestFixture]
+    public class TestAddition
+    {
+        private Addition addition;
+        [SetUp]
+        public void SetUp()
+        {
+            addition = new Addition();
+        }
+        [TestCase(1, 2, 3)]
+        [TestCase(0.1,0.2,0.3)]
+        public void Addding_Positive_Numbers(decimal first, decimal second, decimal result)
```

Comment:
Fix spelling error and maintain consistent method name format/punctuation as with the unit tests for the modulo operator

Reply:
This applies to all tests